src/main/java/org/apache/ofbiz/ofbiztutorial/usermgr/events/UserEvents.java

```diff
@@ -0,0 +1,132 @@
+package org.apache.ofbiz.ofbiztutorial.usermgr.events;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.apache.ofbiz.base.util.Debug;
+import org.apache.ofbiz.base.util.UtilValidate;
+import org.apache.ofbiz.entity.GenericValue;
+import org.apache.ofbiz.service.LocalDispatcher;
+import org.apache.ofbiz.service.ServiceUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UserEvents {
+
+    public static final String MODULE = UserEvents.class.getName();
+
+    public static String createUserEvent(HttpServletRequest request, HttpServletResponse response) {
+        LocalDispatcher dispatcher = (LocalDispatcher) request.getAttribute("dispatcher");
+        // Extract data from the form
+        HttpSession session = request.getSession();
+        GenericValue userLogin = (GenericValue) session.getAttribute("userLogin");
+        System.out.println("userLogin "+userLogin+"========================");
+        String userLoginId = request.getParameter("userLoginId");
+        String password = request.getParameter("password");
+        String confirmPassword = request.getParameter("confirmPassword");
+        String firstName = request.getParameter("firstName");
+        String lastName = request.getParameter("lastName");
+        String middleName = request.getParameter("middleName");
+//        String gender = request.getParameter("gender");
+<<<<<<< HEAD
```

Comment:
you've pushed the conflicts please remove this.

Reply:
Removed these conflicts in the recent commit.